packages/fast-components-react-msft/src/flyout/README.md

```diff
@@ -0,0 +1,6 @@
+# Flyout
+The *flyout* component accepts children and follows the functional pattern of a non-modal dialog (non-modal dialogs allow users the ability to continue interacting with the page). The width and height of the flyout can be set by props. If no props are passed to set width or height, the dialog width will default to 280px and the height will default to 128px.
```

Comment:
nit: "...props are passed to set width or height, the **dialog** width will..." **dialog** should be **flyout** here to avoid confusion I think